src/components/charts/SavedValues/SavedValuesList.tsx

```diff
@@ -75,6 +78,10 @@ export default function SavedValuesList() {
         history.push(`${URLS.CHARTS_DISCOVER}${URLS.CHART}/${chartId}${URLS.PRESET_VALUES}/${chartValueId}`)
     }
 
+    const redirectToDeployChartPage = (chartValueId: number): void => {
```

Comment:
can we reuse redirectToChartValuePage instead of re writing this?

Reply:
Done